core-services/prow/02_config/openshift/cloud-provider-nutanix/_pluginconfig.yaml

```diff
@@ -0,0 +1,13 @@
+approve:
```

Comment:
You need to set up the backport risk assessed and cherry pick approval labels here